pyfr/backends/openmp/kernels/par-memcpy.mako

```diff
@@ -2,16 +2,14 @@
 <%inherit file='base'/>
 <%namespace module='pyfr.backends.base.makoutil' name='pyfr'/>
 
-#include <string.h>
-
 void
-par_memcpy(char *dst, const char *src, int n)
+par_memcpy(fpdtype_t *dst, int dblocksz, const fpdtype_t *src, int sblocksz,
```

Comment:
Is it possible to detect when a straight memcpy would work without needing to do things in a block by block basis?

Reply:
It is, I can compare the blocksz of both arrays and do a straight copy if they match